velox/exec/WindowFunction.h

```diff
@@ -159,8 +170,12 @@ std::optional<std::vector<FunctionSignaturePtr>> getWindowFunctionSignatures(
 struct WindowFunctionEntry {
   std::vector<FunctionSignaturePtr> signatures;
   WindowFunctionFactory factory;
+  ProcessingUnit processingUnit;
 };
 
+std::optional<const WindowFunctionEntry*> getWindowFunctionEntry(
```

Comment:
Is this required to get the ProcessingUnit for a function ? If so, then we can add a function to return specifically that. I feel that we shouldn't expose WindowFunctionFactory to users.

Reply:
@aditi-pandit Yes. We need to check whether support rows streaming window processing in Window.cpp based on the processing unit.  Already added getWindowFunctionMetadata function to get the ProcessingUnit metadata.